TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -231,6 +300,10 @@ namespace Plugin {
     uint32_t TextToSpeechImplementation::SetPrimaryVolDuck(const uint8_t prim)
     {
         _adminLock.Lock();
+
```

Comment:
This will lead to deadlock scenarios. It is now possible to return from SetPrimaryVolDuck() function with _adminLock locked. Next time any API is called, it will lead do deadlock.

Reply:
No need to hold the admin lock here for validation, the lock can be simply moved down